appendices/package-approval-template.md

```diff
@@ -1,11 +1,16 @@
 ```markdown
-----------------------------------------------
+---
+
 🎉 <package-name-here> has been approved by pyOpenSci! Thank you <maintainer-name-here> for submitting <package-name> and many thanks to <reviewer-names-here> for reviewing this package! 😸
 
+## Author Wrap Up Tasks
+
 There are a few things left to do to wrap up this submission:
+
 - [ ] Activate [Zenodo](https://zenodo.org/) watching the repo if you haven't already done so.
 - [ ] Tag and create a release to create a Zenodo version and DOI.
 - [ ] Add the badge for pyOpenSci peer-review to the README.md of <package-name-here>. The badge should be `[![pyOpenSci](https://tinyurl.com/y22nb8up)](https://github.com/pyOpenSci/software-review/issues/issue-number)`.
+- [ ] Please fill out the [post-review survey](https://forms.gle/BLGVCfUdiJHS5YJY7). All maintainers and reviewers should fill this out.
```

Comment:
Are we sure we want this form to be public?

Reply:
Good question. right now the pre-submission form is a part of the peer review submission template. it hasn't been an issue YET... BUT are you thinking we could get spammed? Is there an easier way to provide the links to the authors and editors? i'm very open to ideas!!

Reply:
If it has not been an issue so far, we can only hope it stays that way!
Even though authors and editors are invited to the Slack channel, we saw in previous submissions that this does not guarantee we will not have trouble getting in touch. GH seems to be the safer bet after all

Reply:
ok great. i hear you. let's keep the survey here for now. if we found it becomes a problem in the future we can always adjust in some way.